service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/bucket/CopyGcsBucketDefinitionToControlledStep.java

```diff
@@ -39,15 +41,17 @@
  * CLONED_RESOURCE_DEFINITION is put into the working map for future steps. A
  * CLONE_DEFINITION_RESULT object is put into the working map, and if the cloning instructions are
  * COPY_DEFINITION, the response is set on the flight.
+ *
+ * <p>Keep in sync with CopyGcsBucketDefinitionToReferencedStep.
```

Comment:
Always a red flag when you have to add a comment like that. Can we figure out a way to DRY this out?